### DISCUSSION TITLE
얼굴인식하는 K-WASM 서비스 데모서비스를 추가하였습니다.

### DISCUSSION THREAD

**Discussion by Hyeopgeon-Lee** [K-PaaS-incubator/k-wasm-app#8] [Announcements]
웹에서 동작 가능한 경량화된 얼굴인식 AI 데모서비스를 오픈하였습니다.

https://sample.k-wasm.kr/html/example1/face_detection.html

**Comment by bingguxi** [comment]
좋은 기술이네요.

**Comment by chna8137** [comment]
많이 배우고 갑니다!!

**Comment by Geuniv** [comment]
기술의 발전이 놀랍네요 !

**Comment by Youjeen-Lee** [comment]
도움이 되었습니다! 감사해요

**Comment by chlwnstkd** [comment]
많은 도움 되었습니다!
매번 감사합니다

**Comment by BlueFingers97** [comment]
WOW!! This is really fresh technology!

**Comment by Junsu97** [comment]
WebAssembly가 적용된 서비스네요!
멋있습니다

**Comment by mu9800mu12** [comment]
I frequently tune into educational content and would greatly appreciate if you could continue to provide such materials
